languages/elixir/exercises/concept/numbers/.docs/after.md

```diff
@@ -0,0 +1,114 @@
+There are two different kinds of numbers in Elixir - integers and floats.
+
+## Integers
+
+Integers are whole numbers.
+
+```elixir
+integer = 3
+# => 3
+```
+
+Big integers (but also floats) are usually formatted by separating every 3 digits into groups with an underscore.
```

Comment:
Ooh, this is an excellent idea. I'll add this to the C# and F# numbers exercise's `after.md` document too.